test/index.ts

```diff
@@ -7,6 +7,13 @@ declare global {
     type Optional = { a: number };
 
     type List = { a: number };
+
+    type String = { one: string };
```

Comment:
What's the purpose of creating a type with the same name of `String` built-in class?

Reply:
Sorry, this is actually incorrectly typed, I think. I was using this to test whether defining a type which mapped to a string field but was not defined as a string would error. Will change this and update the type name too